rpc/src/query.rs

```diff
@@ -442,16 +442,31 @@ pub enum Condition {
     Exists(String),
 }
 
+impl Condition {
+    /// Return the key that this condition applies to
+    pub fn key(&self) -> &str {
```

Comment:
Given that a key is required for every condition, would it make sense to perhaps change `Condition` to be a struct that contains a key and a separate enum for the condition variants?

Reply:
As long as we don't foresee a new type of condition which does not apply to a key, then yeah I guess it would make sense. Happy to take care of that.

Reply:
Here's a PR for this, targeting this branch: https://github.com/informalsystems/tendermint-rs/pull/1231